cloudkeeper-one.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'settingslogic', '~> 2.0'
   spec.add_runtime_dependency 'activesupport', '~> 4.0'
   spec.add_runtime_dependency 'tilt', '~> 2.0'
+  spec.add_runtime_dependency 'opennebula', '~> 5.2'
```

Comment:
Does it complain? I've never had this issue and I'm using `~> 5.2`-like dependencies in all my projects.